### PR TITLE
Update zh-cn.json

### PR DESCRIPTION
**Pull request by steve02081504**

### CODE REVIEW

**Comment by Cohee1207** [comment]
This will inevitably conflict with https://github.com/SillyTavern/SillyTavern/pull/2374 as you've both added localization for assets extension.

**Comment by steve02081504** [comment]
> This will inevitably conflict with #2374 as you've both added localization for assets extension.

I've tried merging branches. There's no conflict👍

File: public/scripts/extensions/quick-reply/html/qrEditor.html [hunk 1]
```diff
@@ -59,48 +59,48 @@ <h3>Context Menu</h3>
 		</div>
 
 
-		<h3>Auto-Execute</h3>
+		<h3 data-i18n="Auto-Execute">Auto-Execute</h3>
 		<div class="flex-container flexFlowColumn">
 			<label class="checkbox_label" title="Prevent this quick reply from triggering other auto-executed quick replies while auto-executing (i.e., prevent recursive auto-execution)">
 				<input type="checkbox" id="qr--preventAutoExecute" >
-				<span><i class="fa-solid fa-fw fa-plane-slash"></i> Don't trigger auto-execute</span>
+				<span><i class="fa-solid fa-fw fa-plane-slash"></i><div data-i18n="Don't trigger auto-execute">Don't trigger auto-execute</div></span>
```
Comment (Cohee1207) [inline public/scripts/extensions/quick-reply/html/qrEditor.html]: Should these be div and not span?

File: public/scripts/extensions/vectors/settings.html [hunk 2]
```diff
@@ -178,13 +178,13 @@ <h4>
...
 
-            <h4>
+            <h4 data-i18n="File vectorization settings">
                 File vectorization settings
             </h4>
 
             <label class="checkbox_label" for="vectors_enabled_files">
                 <input id="vectors_enabled_files" type="checkbox" class="checkbox">
-                Enabled for files
+                <div data-i18n="Enable for files">Enable for files</div>
```
Comment (Cohee1207) [inline public/scripts/extensions/vectors/settings.html]: Same as above - prefer span for inline elements

File: public/scripts/extensions/quick-reply/html/qrEditor.html (hunk 1 above)
Comment (steve02081504) [reply public/scripts/extensions/quick-reply/html/qrEditor.html]: fixed by https://github.com/SillyTavern/SillyTavern/pull/2376/commits/3c696c64b341436768746863591250da6c336932

File: public/scripts/extensions/vectors/settings.html (hunk 2 above)
Comment (steve02081504) [reply public/scripts/extensions/vectors/settings.html]: fixed by https://github.com/SillyTavern/SillyTavern/pull/2376/commits/3c696c64b341436768746863591250da6c336932

File: public/scripts/extensions/expressions/settings.html [hunk 3]
```diff
@@ -1,80 +1,81 @@
...
+                        <span data-i18n="Upload sprite pack (ZIP)">Upload sprite pack (ZIP)</span>
                     </div>
                     <div id="expression_override_cleanup_button" class="menu_button">
                         <i class="fa-solid fa-trash-can"></i>
-                        <span>Remove all image overrides</span>
+                        <span data-i18n="Remove all image overrides">Remove all image overrides</span>
                     </div>
                 </div>
-                <p class="hint"><b>Hint:</b> <i>Create new folder in the <b>/characters/</b> folder of your user data directory and name it as the name of the character.
+                <p class="hint" data-i18n="&lt;b&gt;Hint:&lt;/b&gt; &lt;i&gt;Create new folder in the &lt;b&gt;/characters/&lt;/b&gt; folder of your user data directory and name it as the name of the character.
```
Comment (Cohee1207) [inline public/scripts/extensions/expressions/settings.html]: It doesn't make sense to include `<b>` and other tags into localizable texts - tags are escaped on insert.

File: public/scripts/extensions/caption/settings.html [hunk 4]
```diff
@@ -54,36 +54,36 @@
...
+                        <option data-type="llamacpp" value="llamacpp_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="ooba" value="ooba_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="koboldcpp" value="koboldcpp_current" data-i18n="[Currently loaded]">[Currently loaded]</option>
+                        <option data-type="custom" value="custom_current" data-i18n="[Currently selected]">[Currently selected]</option>
                     </select>
                 </div>
                 <label data-type="openai,anthropic,google" class="checkbox_label flexBasis100p" for="caption_allow_reverse_proxy" title="Allow using reverse proxy if defined and valid.">
                     <input id="caption_allow_reverse_proxy" type="checkbox" class="checkbox">
-                    Allow reverse proxy
+                    <div data-i18n="Allow Reverse Proxy">Allow Reverse Proxy</div>
```
Comment (Cohee1207) [inline public/scripts/extensions/caption/settings.html]: Wrong capitalization

File: public/scripts/extensions/caption/settings.html [hunk 5]
```diff
@@ -54,36 +54,36 @@
...
                 </div>
             </div>
             <div id="caption_prompt_block">
-                <label for="caption_prompt">Caption Prompt</label>
+                <label for="caption_prompt" data-i18n="Caption Prompt">Caption Prompt</label>
                 <textarea id="caption_prompt" class="text_pole" rows="1" placeholder="&lt; Use default &gt;">${PROMPT_DEFAULT}</textarea>
                 <label class="checkbox_label margin-bot-10px" for="caption_prompt_ask" title="Ask for a custom prompt every time an image is captioned.">
                     <input id="caption_prompt_ask" type="checkbox" class="checkbox">
-                    Ask every time
+                    <div data-i18n="Ask every time">Ask every time</div>
```
Comment (Cohee1207) [inline public/scripts/extensions/caption/settings.html]: Replace div with span

File: public/scripts/extensions/quick-reply/html/qrEditor.html [hunk 6]
```diff
@@ -59,48 +59,48 @@ <h3>Context Menu</h3>
...
 		</div>
 
 
-		<h3>Testing</h3>
+		<h3 data-i18n="Testing">Testing</h3>
 		<div id="qr--modal-executeButtons">
 			<div id="qr--modal-execute" class="qr--modal-executeButton menu_button" title="Execute the quick reply now">
 				<i class="fa-solid fa-play"></i>
-				Execute
+				<div data-i18n="Execute">Execute</div>
```
Comment (Cohee1207) [inline public/scripts/extensions/quick-reply/html/qrEditor.html]: Replace div with span

File: public/scripts/extensions/quick-reply/html/settings.html [hunk 7]
```diff
@@ -1,22 +1,22 @@
...
 	<div class="inline-drawer">
 		<div class="inline-drawer-toggle inline-drawer-header">
-			<strong>Quick Reply</strong>
+			<strong data-i18n="Quick Reply">Quick Reply</strong>
 			<div class="inline-drawer-icon fa-solid fa-circle-chevron-down down"></div>
 		</div>
 		<div class="inline-drawer-content">
 			<label class="flex-container">
-				<input type="checkbox" id="qr--isEnabled"> Enable Quick Replies
+				<input type="checkbox" id="qr--isEnabled"><div data-i18n="Enable Quick Replies">Enable Quick Replies</div>
```
Comment (Cohee1207) [inline public/scripts/extensions/quick-reply/html/settings.html]: Replace div with span

File: public/scripts/extensions/vectors/settings.html [hunk 8]
```diff
@@ -132,33 +132,33 @@
...
 
-            <h4>
+            <h4 data-i18n="World Info settings">
                 World Info settings
             </h4>
 
             <label class="checkbox_label" for="vectors_enabled_world_info" title="Enable activation of World Info entries based on vector similarity.">
                 <input id="vectors_enabled_world_info" type="checkbox" class="checkbox">
-                Enabled for World Info
+                <div data-i18n="Enable for World Info">Enable for World Info</div>
```
Comment (Cohee1207) [inline public/scripts/extensions/vectors/settings.html]: Replace div with span

File: public/scripts/extensions/vectors/settings.html [hunk 9]
```diff
@@ -196,152 +196,153 @@ <h4>
...
+                                <option value="1" data-i18n="User">User</option>
+                                <option value="2" data-i18n="Assistant">Assistant</option>
                             </select>
                         </label>
                     </div>
                 </div>
                 <div class="flex-container">
                     <div id="vectors_files_vectorize_all" class="menu_button menu_button_icon" title="Vectorize all files in the Data Bank and current chat.">
-                        Vectorize All
+                        <div data-i18n="Vectorize All">Vectorize All</div>
```
Comment (Cohee1207) [inline public/scripts/extensions/vectors/settings.html]: Replace div with span

File: public/scripts/extensions/expressions/settings.html (hunk 3 above)
Comment (steve02081504) [reply public/scripts/extensions/expressions/settings.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/9414c25edac4302f42690fdefaabd95e5ad64ce7

File: public/scripts/extensions/caption/settings.html (hunk 4 above)
Comment (steve02081504) [reply public/scripts/extensions/caption/settings.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7

File: public/scripts/extensions/quick-reply/html/qrEditor.html (hunk 6 above)
Comment (steve02081504) [reply public/scripts/extensions/quick-reply/html/qrEditor.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7

File: public/scripts/extensions/caption/settings.html (hunk 5 above)
Comment (steve02081504) [reply public/scripts/extensions/caption/settings.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7

File: public/scripts/extensions/quick-reply/html/settings.html (hunk 7 above)
Comment (steve02081504) [reply public/scripts/extensions/quick-reply/html/settings.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7

File: public/scripts/extensions/vectors/settings.html (hunk 8 above)
Comment (steve02081504) [reply public/scripts/extensions/vectors/settings.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7

File: public/scripts/extensions/vectors/settings.html (hunk 9 above)
Comment (steve02081504) [reply public/scripts/extensions/vectors/settings.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/12c5466f08555ea88e350ba5aff0572878e44ba7

File: public/scripts/extensions/caption/settings.html [hunk 10]
```diff
@@ -54,36 +54,36 @@
...
+                        <option data-type="llamacpp" value="llamacpp_current" data-i18n="currently_loaded">[Currently loaded]</option>
+                        <option data-type="ooba" value="ooba_current" data-i18n="currently_loaded">[Currently loaded]</option>
+                        <option data-type="koboldcpp" value="koboldcpp_current" data-i18n="currently_loaded">[Currently loaded]</option>
+                        <option data-type="custom" value="custom_current" data-i18n="currently_selected">[Currently selected]</option>
                     </select>
                 </div>
                 <label data-type="openai,anthropic,google" class="checkbox_label flexBasis100p" for="caption_allow_reverse_proxy" title="Allow using reverse proxy if defined and valid.">
                     <input id="caption_allow_reverse_proxy" type="checkbox" class="checkbox">
-                    Allow reverse proxy
+                    <span data-i18n="Allow reverse Proxy">Allow reverse Proxy</span>
```
Comment (Cohee1207) [inline public/scripts/extensions/caption/settings.html]: Sorry but capitalization is still changed from the original

Comment (steve02081504) [reply public/scripts/extensions/caption/settings.html]: fixed in https://github.com/SillyTavern/SillyTavern/pull/2376/commits/c67407e80b74d82d129a80140bfcc49ca76cce9d
ty for help!

**Pull request merged**